### PR TITLE
[NIO] Inconsistent failure messages fix

### PR DESCRIPTION
**Pull request by vam-google**
https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2090

Fixing only the example case, mentioned in the issue description.

### CODE REVIEW

**Comment by vam-google** [comment]
Requesting review from @garrettjonesgoogle, as I could not add @Horneth as a reviewer

File: google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java [hunk 1]
```diff
@@ -572,7 +572,7 @@ public void checkAccess(Path path, AccessMode... modes) throws IOException {
     // null size indicate a file that we haven't closed yet, so GCS treats it as not there yet.
     if (null == blobInfo || blobInfo.getSize() == null) {
       throw new NoSuchFileException(
-          cloudPath.getBlobId().getBucket() + "/" + cloudPath.getBlobId().getName());
+          "gs://" + cloudPath.getBlobId().getBucket() + "/" + cloudPath.getBlobId().getName());
```
Comment (garrettjonesgoogle) [inline google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java]: Why not use `String.format` like the other referenced error message?

https://github.com/GoogleCloudPlatform/google-cloud-java/blob/f72746c6183378664a9326ee406d679926a7def4/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageReadChannel.java#L196

**Comment by Horneth** [comment]
@vam-google Looks good to me 👍  Thanks !

File: google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java (hunk 1 above)
Comment (vam-google) [inline google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java]: String.format() is locale-specific, I prefer not using String.format() when it is not purely about human readable messages without technical details (like path) in it. This is inspired by this issue: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1957

On different locales the outputs of string.format may be unpredictable.

**Comment by vam-google** [comment]
Closing as a duplicate of an already merged PR https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2330

**Pull request closed without merging**